app/assets/javascripts/controllers/topology/middleware_topology_controller.js

```diff
@@ -1,42 +1,25 @@
 angular.module('ManageIQ').controller('middlewareTopologyController', MiddlewareTopologyCtrl);
-MiddlewareTopologyCtrl.$inject = ['$scope', '$http', '$interval', '$location', 'topologyService', 'miqService'];
+MiddlewareTopologyCtrl.$inject = ['$scope', '$interval', 'topologyService'];
 
-function MiddlewareTopologyCtrl($scope, $http, $interval, $location, topologyService, miqService) {
+function MiddlewareTopologyCtrl($scope, $interval, topologyService) {
   ManageIQ.angular.scope = $scope;
   miqHideSearchClearButton();
   var vm = this;
   vm.vs = null;
   var d3 = window.d3;
   vm.d3 = d3;
-  var icons;
+  vm.icons;
```

Comment:
Useless property access? :)

Reply:
icons are loaded in services in `mixinRefresh`. This function is also used for initial data loading.